src/MeetingsSDKAdapter.js

```diff
@@ -509,6 +588,154 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     return concat(getDisplayData$, localMediaUpdateEvent$);
   }
 
+  /**
+   * Attempts to start/stop screen sharing to the given meeting ID.
+   * If successful, a sharing start/stop event is dispatched.
+   *
+   * @param {string} ID ID of the meeting to start/stop sharing
+   * @memberof MeetingJSONAdapter
+   */
+  async handleLocalShare(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    if (!sdkMeeting.canUpdateMedia()) {
+      // eslint-disable-next-line no-console
+      console.error(`Unable to update screen share for meeting "${ID}" due to unstable connection.`);
+
+      return;
+    }
+
+    const enableSharingStream = async () => {
+      const [, localShare] = await sdkMeeting.getMediaStreams({sendShare: true});
+
+      this.meetings[ID].localShare = localShare;
+
+      // Due to SDK limitation around local media updates,
+      // we need to emit a custom event for video mute updates
+      sdkMeeting.emit(EVENT_MEDIA_LOCAL_UPDATE, {
+        control: SHARE_CONTROL,
+        state: MeetingControlState.DISABLED,
+      });
+
+      await sdkMeeting.updateShare({stream: localShare, sendShare: true, receiveShare: true});
+    };
+
+    const disableSharingStream = async () => {
+      sdkMeeting.emit(EVENT_MEDIA_LOCAL_UPDATE, {
+        control: SHARE_CONTROL,
+        state: MeetingControlState.INACTIVE,
+      });
+
+      await sdkMeeting.updateShare({
+        sendShare: false,
+        receiveShare: true,
+      });
+
+      this.stopStream(this.meetings[ID].localShare);
+      this.meetings[ID].localShare = null;
+    };
+
+    const resetSharingStream = (error) => {
+      // eslint-disable-next-line no-console
+      console.error(`Unable to update local share for meeting "${ID}"`, error);
```

Comment:
We should change this to the SDK logger. the component should be logging it's own debuggability information

Reply:
ok, just updated. Thanks!